crates/holochain_integrity_types/src/capability/grant.rs

```diff
@@ -16,6 +16,7 @@ use std::collections::BTreeSet;
 ///
 /// See `.is_valid()`
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq)]
+#[serde(tag = "type", content = "value", rename_all = "snake_case")]
 #[allow(clippy::large_enum_variant)]
 pub enum CapGrant {
```

Comment:
This isn't safe to return over the admin interface because it may contain a `CapAccess` which may contain a `CapSecret`. This is one where it's not just cleaner to return a different type when we implement listing grants, it's actually a security question what we should be exposing.

This looks to me like it isn't an input type on the `GrantZomeCallCapabilityPayload`?

Reply:
> This looks to me like it isn't an input type on the GrantZomeCallCapabilityPayload

Yes, that's correct.

Reply:
> This is one where it's not just cleaner to return a different type when we implement listing grants, it's actually a security question what we should be exposing

I don't understand what you mean there exactly. As a tangential note, I think I changed this enum because [it showed up in the js-client types](https://github.com/holochain/holochain-client-js/blob/main/src/hdk/capabilities.ts#L66), probably because it's exposed in the hdk (I actually don't know why it's exposed, I can only find ways to retrieve one's own `ZomeCallCapGrant`s as part of private `Entry`s but not how to retrieve a `CapGrant` type).

Reply:
So I'm going to remove this change again....